config/apidoc.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+return [
+
+    /*
+     * The output path for the generated documentation.
+     * This path should be relative to the root of your application.
+     */
+    'output' => 'docs',
+
+    /*
+     * The router to be used (Laravel or Dingo).
+     */
+    'router' => 'laravel',
+
+    'faker_seed' => 0,
```

Comment:
Yeah, so the seed can't be 0... cuz php:
[mpociot/laravel-apidoc-generator/blob/master/src/Tools/Generator.php#L316](https://github.com/mpociot/laravel-apidoc-generator/blob/master/src/Tools/Generator.php#L316)